internal/operators/manager_test.go

```diff
@@ -27,6 +32,59 @@ import (
 	"sigs.k8s.io/yaml"
 )
 
+func MatchControllerManifest(expectedName, expectedDecodedBodyRegexp string) gomock.Matcher {
+	return controllerManifestMatcher{
+		expectedName,
+		expectedDecodedBodyRegexp,
+	}
+}
+
+func decodeInterface(x interface{}) ([]map[string]string, error) {
+	data := []map[string]string{}
+	jsonContentBytes, ok := x.([]byte)
+	if !ok {
+		return data, errors.New("wtf")
```

Comment:
While I understand this is very unlikely to happen perhaps create a more informative error string :sweat_smile:

Reply:
Ooops I didn't notice I've left this in :sweat_smile: